### PR TITLE
written ResumeParserAPI test cases

### PR DESCRIPTION
**Pull request by AngadCel**
written ResumeParserAPI test cases

### CODE REVIEW

File: cypress/e2e/ResumeParserAPI/pricing.cy.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+    });
+
+    it('2. Test to check "Resume Parser API" title is present', () => {
+        cy.componentVisiblityCheck('h1, h2, h3, h4', 'Resume Parser API');
+    });
+
+    // Test to check visibility of the "Free Plan" title and its price
+    it('3. Test to check "Free Plan" title and price are present', () => {
+        // Verify "Free Plan" title visibility and content
+        cy.get('.card-header .d-block.h3')
```
Comment (sinchanagoudar) [inline cypress/e2e/ResumeParserAPI/pricing.cy.js:20]: test case is missing to check  
image

File: cypress/e2e/ResumeParserAPI/pricing.cy.js [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+        // Verify the "Starter Plan" title is visible and contains the correct text
+        cy.get('span.d-block.h3')
+            .should('be.visible')
+            .and('contain.text', 'Starter Plan');
+
+        // Verify the price of the "Starter Plan" is visible and contains the correct amount
+        cy.get('.plan-price.font-size-4.font-weight-bold.text-primary')
+            .should('be.visible')  // Ensure the price element is visible
+            .and('contain.text', '$29.99'); // Verify the price text is correct
+    });
```
Comment (sinchanagoudar) [inline cypress/e2e/ResumeParserAPI/pricing.cy.js:96]: Test case is missing to check in the starter plan
image

File: cypress/e2e/ResumeParserAPI/apiInfo.cy.js [hunk 3]
```diff
@@ -0,0 +1,211 @@
...
+    });
+
+    it('11. Test to check the shell command snippet is present', () => {
+        // Check if the <pre> element with class "language-shell" is present
+        cy.get('pre.language-shell').should('exist');
+    });  
+    
+    it("12. Test to check visibility of 'Bulk processing of resumes' heading", () => {
+        // Check if the <h4> heading is present
+        cy.get('article > :nth-child(20)').should('exist');
```
Comment (sinchanagoudar) [inline cypress/e2e/ResumeParserAPI/apiInfo.cy.js:75]: seeing nth child can you find alternative

File: cypress/e2e/ResumeParserAPI/apiInfo.cy.js [hunk 4]
```diff
@@ -0,0 +1,211 @@
...
+    
+        // Optionally check if the heading text is exactly as expected
+        cy.get('article > :nth-child(20)').should('have.text', 'Bulk processing of resumes');
```
Comment (sinchanagoudar) [inline cypress/e2e/ResumeParserAPI/apiInfo.cy.js:78]: remoce nth child

File: cypress/e2e/ResumeParserAPI/apiInfo.cy.js [hunk 5]
```diff
@@ -0,0 +1,211 @@
...
+    });
+    
+    
+    it('13. Test to check the presence of "Security and Privacy" heading', () => {
+        // Check if the <h4> element with the specified text is present
+        cy.get('article > :nth-child(23)').should('exist');
+    
+        // Optionally, check if the text is exactly as expected
+        cy.get('article > :nth-child(23)').should('have.text', 'Security and Privacy');
```
Comment (sinchanagoudar) [inline cypress/e2e/ResumeParserAPI/apiInfo.cy.js:87]: used to many time nth child find alternative allover the page

File: cypress/e2e/ResumeParserAPI/apiInfo.cy.js (hunk 3 above)
Comment (AngadCel) [reply cypress/e2e/ResumeParserAPI/apiInfo.cy.js:75]: i didnt got any alternative s i am getting class  from cypress and using it

File: cypress/e2e/ResumeParserAPI/apiInfo.cy.js (hunk 4 above)
Comment (AngadCel) [reply cypress/e2e/ResumeParserAPI/apiInfo.cy.js:78]: i didnt got any alternative s i am getting class  from cypress and using it

File: cypress/e2e/ResumeParserAPI/apiInfo.cy.js (hunk 5 above)
Comment (AngadCel) [reply cypress/e2e/ResumeParserAPI/apiInfo.cy.js:87]: i didnt got any alternative s i am getting class  from cypress and using it

File: cypress/e2e/ResumeParserAPI/pricing.cy.js (hunk 2 above)
Comment (AngadCel) [reply cypress/e2e/ResumeParserAPI/pricing.cy.js:96]: done

File: cypress/e2e/ResumeParserAPI/pricing.cy.js (hunk 1 above)
Comment (AngadCel) [reply cypress/e2e/ResumeParserAPI/pricing.cy.js:20]: done

**Pull request merged**